client/src/pages/cases/components/TimelineDetail.scss

```
@@ -0,0 +1,78 @@
+@import 'node_modules/govuk-frontend/govuk/base';
+
+.container {
+  display: flex;
+  flex-direction: row;
+  padding: 0.5rem 0px;
```

Try not to mix units like this. Zero values don't need units anyway. 🙂 
```suggestion
  padding: 0.5rem 0;
```

---

Noted